mysql-test/suite/tokudb/r/simple_join_tokudb_innodb.result

```diff
@@ -18,13 +18,13 @@ b	c
 3	9
 3	27
 4	1
-select a,tab.b,c from tab join tbc on tab.b = tbc.b;
+select a,tab.b,c from tab join tbc on tab.b = tbc.b order by a, tab.b, c;
```

Comment:
something in 8.0.18 that needs order by now?

Reply:
Without 'order by' 8.0.18 returns the same set of rows, but order is different than in 8.0.17